### DISCUSSION TITLE
Generalize area=yes

### DISCUSSION THREAD

**Discussion by yvecai** [osmlab/osm-data-model#30] [Study/Steps]
I'm always amazed in wiki tag definitions stating "Rendered as area if first and last point are the same."
Transitioning to an area data type could be easier if we would first tag explicitly area=yes on existing ways (and relations?).
This is a big amount of work but a simple one. And yes, a silly one. This would expose the internal rules and recipes used by various softwares to guess which element is an area well before an API change and allow us to deal with corner-cases before such jump. 
But firstly this would be a way to onboard the community at large with the introduction of an area type. Data model change can be complex to understand, but such a re-mapping campaign is not. 

From this, area validation can evolve and be tested at whatever level desired (editors, API) without the overhead of a bigger data model change.

**Comment by 501Ghost** [comment]
If we change to a system with something that resembles the point, lines and polygons that other GIS software has, then `area=yes` would be implicit for the polygons. Unclosed ways like roundabouts will then be treated as lines and not as polygons.

Is it technically necessary to tag `area=yes` explicitly in OSM when/before this update takes place? Are there other ways to do this?
